awsmp/cli.py

```diff
@@ -22,8 +23,30 @@
 
 
 @click.group()
-def cli():
-    pass
+@click.option("--log-console/--no-log-console", default=False, is_flag=True, help="Do not log to console")
+@click.option("--log-file", required=False, help="File to log to", type=click.Path(dir_okay=False))
+@click.option("-v", "--verbose", count=True, help="Set verbosity level. May supply multiple times")
```

Comment:
nit: I prefer to just set the log level. otherwise you never know what `-vv` means vs `-vvv` vs. `-vvvvvv`, ...

Reply:
I'm personally fine with `-v` as it's a normal pattern. That said, having a `--debug` flag is just more descriptive and, at least personally, I usually want standard or all logs, seldom do I want middle ground